### ISSUE TITLE
[office-addin-manifest] issues installing & running

### ISSUE THREAD

**Issue by SeanKilleen** [OfficeDev/Office-Addin-Scripts#217]
Hey all, 

Thanks for putting a package together to assist in the validation of a manifest.

Not sure if this is an issue with my install or what but the `office-addin-manifest` package does not appear to work out of the box for me. Exact steps below.

Version check + install

```
C:\Users\SeanK> node -v
v10.16.3
C:\Users\SeanK> npm -v
6.9.0
C:\Users\SeanK> npm install -g office-addin-manifest
C:\Program Files\nodejs\office-addin-manifest -> C:\Program Files\nodejs\node_modules\office-addin-manifest\cli.js
+ office-addin-manifest@1.4.12
```

So far so good. I re-open the window but then I can't get it to tell me what version it is:

```
C:\Users\SeanK> office-addin-manifest
Usage: office-addin-manifest [options] [command]

Options:
  -V, --version                     output the version number
  -h, --help                        output usage information

Commands:
  info <manifest-path>
  modify [options] <manifest-path>
  validate <manifest-path>
C:\Users\SeanK> office-addin-manifest -V
(version not available)
C:\Users\SeanK> office-addin-manifest -v
error: unknown option `-v'
```

And when I validate a manifest that I copied from the docs I can't get the tool to provide a response:

```
C:\Users\SeanK> office-addin-manifest validate C:\Users\SeanK\Repositories\[Redacted]\AppManifest.xml
Error: The manifest validation service did not return the expected response.
```

I pulled up Fiddler to see if any traffic was going out to a validation service but that doesn't appear to be the case.

## Version info

To recap the above: 

* Winver: Windows 10 Build `1903` (`18362.356`)
* Node: 10.16.3
* Npm: 6.9.0
* office-addin-manifest: 1.4.2.

**Comment by SeanKilleen** [comment]
Ah, I figured it out! An errant comment in my XML at the top that was the issue (I guess it wasn't an exact copy from the docs! 😆).

It would be helpful if there was more verbose error output in cases like this, given that I'd imagine this would be a case of an invalid manifest.

**Comment by akrantz** [comment]
Thanks for reporting the issue. I was able to add a comment on the top line and verify the issue. We'll need to investigate the validation service response to see why this doesn't get reported better.

**Comment by SeanKilleen** [comment]
@LouMM  is there a reason this was closed with no resolution after it was reproduced?

**Comment by TCourtneyOwen** [comment]
Perhaps this was mistakenly closed during our issues triage today. I will reactivate

**Comment by LouMM** [comment]
Passing this along to our Store team for review on better error handling during validation. Thank you for the suggestions @SeanKilleen
